mlbusinesscomponents/src/main/java/com/mercadolibre/android/mlbusinesscomponents/components/touchpoint/view/flex_cover_carousel/FlexCoverCarouselComponent.kt

```diff
@@ -0,0 +1,98 @@
+package com.mercadolibre.android.mlbusinesscomponents.components.touchpoint.view.flex_cover_carousel
+
+import android.content.Context
+import android.util.AttributeSet
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.PagerSnapHelper
+import androidx.recyclerview.widget.RecyclerView
+import com.mercadolibre.android.mlbusinesscomponents.R
+import com.mercadolibre.android.mlbusinesscomponents.components.touchpoint.callback.OnClickCallback
+import com.mercadolibre.android.mlbusinesscomponents.components.touchpoint.domain.model.AdditionalEdgeInsets
+import com.mercadolibre.android.mlbusinesscomponents.components.touchpoint.domain.model.flex_cover_carousel.FlexCoverCard
+import com.mercadolibre.android.mlbusinesscomponents.components.touchpoint.domain.model.flex_cover_carousel.FlexCoverCarouselResponse
+import com.mercadolibre.android.mlbusinesscomponents.components.utils.ScaleUtils
+
+class FlexCoverCarouselComponent(
+    context: Context,
+    attr: AttributeSet?
+) : RecyclerView(context, attr), FlexCoverCarouselComponentInterface {
```

Comment:
Siempre en las vistas hay que agregar el `@JvmOverloads constructor` porque si no puede traer problemas.
```suggestion
class FlexCoverCarouselComponent @JvmOverloads constructor(
    context: Context,
    attr: AttributeSet? = null,
    defStyle: Int = 0
) : RecyclerView(context, attr, defStyle), FlexCoverCarouselComponentInterface {
```